cli/listener/src/main.rs

```diff
@@ -12,22 +15,23 @@ use solana_sdk::{
     pubkey::Pubkey,
     signature::Keypair,
 };
-use solido_cli_common::snapshot::{
-    Config, OutputMode, SnapshotClient, SnapshotConfig, SnapshotError,
+use solido_cli_common::{
+    error::{AsPrettyError, Error},
+    prometheus::{write_metric, Metric, MetricFamily},
+    snapshot::{Config, OutputMode, SnapshotClient, SnapshotClientConfig},
 };
+use tiny_http::{Header, Request, Response, Server};
 
 #[derive(Clap, Debug)]
 pub struct Opts {
+    /// Solido's instance address
+    #[clap(long)]
+    solido_address: Pubkey,
+
     /// URL of cluster to connect to (e.g., https://api.devnet.solana.com for solana devnet)
-    // Overwritten by `GeneralOpts` if None.
     #[clap(long, default_value = "http://127.0.0.1:8899")]
     cluster: String,
 
-    /// Whether to output text or json. [default: "text"]
-    // Overwritten by `GeneralOpts` if None.
-    #[clap(long = "output", possible_values = &["text", "json"])]
-    output_mode: Option<OutputMode>,
-
     /// Unique name for identifying
     #[clap(long, default_value = "solido")]
```

Comment:
Why do we pass this in? I think it would be better to hard-code, the code path for fetching it is already specific to Solido, making what we insert into the database variable only results in incompatibilities.